c2rust-transpile/src/c_ast/mod.rs

```diff
@@ -536,7 +536,6 @@ impl TypedAstContext {
                 CDeclKind::Function {
                     body: Some(_),
                     is_global: true,
-                    is_inline: false,
```

Comment:
I think the check should become something like `!is_inline || (is_inline && is_extern)`.

Reply:
Although I'm not 100% sure even that is correct.